meilisearch_python_sdk/_client.py

```diff
@@ -144,7 +143,7 @@ def __init__(
         api_key: str | None = None,
         *,
         timeout: int | None = None,
-        verify: str | bool | SSLContext = True,
+        verify: str | bool,
```

Comment:
Instead of removing `SSLContext` we need to remove `str`. We also need to keep the default value of `True` (This should fix your CI failure). 

```suggestion
        verify: bool | SSLContext = True,
```